### PR TITLE
data:CodeKisku

### PR DESCRIPTION
**Pull request by CodeKisku**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8133"></a>

### CODE REVIEW

File: data/CodeKisku.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+      "name": "Follow on Leetcode",
+      "url": "https://leetcode.com/sanatankisku/",
+      "icon": "SiLeetcode"
+    },
+    {
+      "name": "Follow on Twitter",
+      "url": "https://twitter.com/KiskuTweets",
+      "icon": "FaTwitter"
+    },
```
Comment (Vishrut19) [inline data/CodeKisku.json:32]: `,` is not needed

File: data/CodeKisku.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Sanatan Kisku",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a Frontend Developer, MCA 2022 Graduate",
+  "avatar": "https://github.com/CodeKisku.png",
```
Comment (Vishrut19) [inline data/CodeKisku.json:6]: `avatar` is not needed

File: data/CodeKisku.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Sanatan Kisku",
+  "displayStatsPublic": true,
+  "type": "personal",
```
Comment (Vishrut19) [inline data/CodeKisku.json:4]: `displayStatsPublic` and `type` is not needed

**Review by Vishrut19 — CHANGES_REQUESTED** [review]

File: data/CodeKisku.json (hunk 3 above)
Comment (SaraJaoude) [inline data/CodeKisku.json:4]: the following is not required:
```suggestion
```

File: data/CodeKisku.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Sanatan Kisku",
+  "bio": "I am a Frontend Developer, MCA 2022 Graduate",
+  "avatar": "https://github.com/CodeKisku.png",
```
Comment (SaraJaoude) [inline data/CodeKisku.json:4]: not required:
```suggestion
  "bio": "I am a Frontend Developer, MCA 2022 Graduate",
```

File: data/CodeKisku.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "name": "Follow on Leetcode",
+      "url": "https://leetcode.com/sanatankisku/",
+      "icon": "SiLeetcode"
+    },
+    {
+      "name": "Follow on Twitter",
+      "url": "https://twitter.com/KiskuTweets",
+      "icon": "FaTwitter"
+    },
```
Comment (SaraJaoude) [inline data/CodeKisku.json:29]: remove comma:
```suggestion
    }
```

**Comment by SaraJaoude** [comment]
@Vishrut19 thank you for the comments - you may wish to consider making these as inline suggestions moving forward so it is easier for either the maintainer or author to accept them.

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes for the JSON to pass and I will now merge this PR.

**Comment by Vishrut19** [comment]
> @Vishrut19 thank you for the comments - you may wish to consider making these as inline suggestions moving forward so it is easier for either the maintainer or author to accept them.

okay thanks @SaraJaoude will do the same from now onwards.

**Pull request merged**